src/_includes/assets/css/screen.css

```diff
@@ -2473,4 +2473,7 @@ a.banner:hover span {
 
 .rtl-layout .post-full-content p code {
   word-break: normal;
+  direction: ltr;
+  white-space: nowrap;
```

Comment:
```suggestion
```
This would affect the longer code elements and push them off the screen.
<img width="479" alt="Screen Shot 2022-03-21 at 11 40 16 AM" src="https://user-images.githubusercontent.com/4591597/159228163-85a560ac-3ba2-4fda-8efc-a20b9891a999.png">

Here it is without the nowrap:
<img width="478" alt="Screen Shot 2022-03-21 at 11 40 27 AM" src="https://user-images.githubusercontent.com/4591597/159228222-a9cf865e-0807-4712-a956-42e1781493b8.png">